src/components/TextArea.jsx

```diff
@@ -0,0 +1,35 @@
+import { Button, Grid, TextField } from '@mui/material';
+import TextareaAutosize from '@mui/material/TextareaAutosize';
+
+export default function TextArea() {
```

Comment:
This component is not containing just the TextArea so rename it to something more explicit like `InputsBox`